doc/changelog.rst

```diff
@@ -93,6 +93,8 @@ Unavoidable breaking changes
 - Since we are now using ``hatch`` as our build backend, we no longer have a usable ``setup.py`` file
   and require installation using ``pip``.  Attempts to invoke the ``setup.py`` file will raise an exception.
   Additionally, ``pip`` >= 21.3 is now required for editable installs.
+- We no longer support the ``srv`` extra, since ``dnspython`` is included as a dependency in PyMongo 4.7+.
+  Instead of ``pip install pymongo[srv]``, use ``pip install pymongo``.
```

Comment:
Can you add the same for "tls"?

Reply:
The tls extra was only valid for Python 2 which we dropped in PYTHON-2133 pymongo 4.0.

Reply:
Updated